apps/namehashlabs.org/components/2 - molecules/footer.tsx

```diff
@@ -160,10 +157,7 @@ export const Footer = (props: React.HTMLAttributes<HTMLDivElement>) => {
               <span className="text-[#EF4444] mx-1">{"❤️"}</span>
               by
             </span>
-            <Link
-              className="text-black underline xSmall:underline-offset-[4px] xSmall:transition-all xSmall:duration-200 xSmall:hover:underline-offset-[2px]"
-              href="/"
-            >
+            <Link className="!text-sm" href="/">
```

Comment:
We want this link to get more visual attention. Therefore it should have the underline style. For example: https://storybook.namekit.io/?path=/story/ui-link--underline-medium